src/tests/ftest/container/rebuild_tests.yaml

```diff
@@ -0,0 +1,29 @@
+hosts:
+ test_machines:
+  - boro-A
+  - boro-B
+  - boro-C
+  - boro-D
+  - boro-E
+  - boro-F
+timeout: 700
+server:
+ server_group: daos_server
+testparams:
+ ranks: !mux
```

Comment:
I don't know that this adds value as a variable parameter.

Reply:
Removed.